src/System.Management.Automation/engine/Utils.cs

```diff
@@ -210,7 +210,7 @@ internal static BigInteger ParseBinary(ReadOnlySpan<char> digits, bool unsigned)
             // or for lengths < 8 (less than one byte) e.g., 010.
             // Walk right to left, stepping one whole byte at a time (if there are any whole bytes).
             int blockWalker;
-            for (blockWalker = digits.Length - 1; blockWalker >= 7; blockWalker -= 8)
+            for (blockWalker = digits.Length - 1; blockWalker >= 8; blockWalker -= 8)
```

Comment:
This change from 7 to 8?  In the event of a multiple-of-8 length, this won't process the last block in this loop, and will invoke the slower partial-byte loop, to process an entire byte. Should be a correct result still, but slightly slower, wouldn't it?

Reply:
Didn't you run into out of bounds issues with the data when running with 7?  At least, that's what I thought...

Reply:
RIght, no, that was a different thing. Derp. :D